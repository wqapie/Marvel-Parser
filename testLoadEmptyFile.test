# Test loading an empty file

LoadGraph empty empty.tsv

ListNodes empty