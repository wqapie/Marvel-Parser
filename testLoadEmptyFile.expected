# Test loading an empty file

loaded graph empty

empty contains: